src/data/full-quiz.ts

```diff
@@ -7196,5 +7196,49 @@ const fullQuiz = [
       "Waterfall is an opposing project management methodology that has little to no room for feedback or iteration until the product is completely built and tested.",
     Link: "https://www.freecodecamp.org/news/agile-methods-and-methodology-for-beginners/",
   },
+  {
+    Question:
+      "In a Scrum team, what is a Scrum Master?",
+    Answer: "A servant leader whose role is to support the rest of the team and the organization",
+    Distractor1: "A project manager who's main role is to ensure the Developers deliver in time",
+    Distractor2: "An external resource temporarily assigned to the team to audit their work",
+    Distractor3: "Scrum Master is not a role officially recognized by the Scrum guide",
+    Explanation:
+      "The Scrum Master is a leader who supports their colleagues in their activities and does more than just assign tasks."
```

Comment:
The automated check for the linter failed because we are missing a comma at the end

Reply:
I know, fixed and committed.